Octokit/Http/Connection.cs

```diff
@@ -136,6 +136,30 @@ public Connection(ProductHeaderValue productInformation, Uri baseAddress, ICrede
             _jsonPipeline = new JsonHttpPipeline();
         }
 
+        /// <summary>
+        /// Gets the latest API Info - this will be null if no API calls have been made
+        /// </summary>
+        /// <returns><seealso cref="ApiInfo"/> representing the information returned as part of an Api call</returns>
+        public ApiInfo LastApiInfo
+        { 
+            get
+            {
+                lock (LastApiInfoLocker)
+                {
+                    return _lastApiInfo;
+                }
+            }
+            private set
+            {
+                lock (LastApiInfoLocker)
+                {
+                    _lastApiInfo = value;
+                }
+            }
+        }
+        private ApiInfo _lastApiInfo;
+        private readonly object LastApiInfoLocker = new object();
+
```

Comment:
@khellang @Haacked I'm easy either way - with locking/ without

What's the preferred method of resolving coding decisions?  A well argued debate between respectful & collaborate peers?  Rocks/ Paper/ Scissors?  A fight to the death?

(is it wrong to hope for the last one)